docs/howto/edit-in-protege.md

```diff
@@ -0,0 +1,94 @@
+# Editing a term in protege
+
+Before you start:
+
+-   make sure you are working on a branch 
+-   make sure you have the editor's file open in Protege (in ODK ontologies, located in: ./src/ontology/ONT-edit.owl)
+
+## Adding, editing, annotating and deleting axioms
+
+#### Adding annotations 
+
+Using Protégé you can add annotations such as labels, definitions, synonyms, database cross references (dbxrefs) to any OWL entity. The panel on the right, named Annotations, is where these annotations are added. CL includes a pre-declared set of annotation properties. The most commonly used annotations are below. 
```

Comment:
- at this point Protégé has been written as protege, Protege and Protégé. Should we always use the latter for consistency?
- "... synonyms **and** database cross references (dbxrefs)... "

Reply:
it should all be Protégé but honestly that's a pain to put in everytime, so my plan was to do a mass search of protege and switch it to Protégé eventually. It's here now cause this is a full copy and paste from another document.